### PR TITLE
Auto upgrade case for egressip and egressfirewall

### PR DESCRIPTION
**Pull request by huiran0826**
Auto OCP-44316 OCP-44315

Logs:
https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/Runner-v3-smoke/3450/console
https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/Runner-v3-smoke/3451/console

https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/Runner-v3-smoke/3454/console
https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/Runner-v3-smoke/3455/console

@openshift/team-sdn-qe

### CODE REVIEW

**Comment by zhaozhanqi** [comment]
@geliu2016 Could you help review this?

File: features/upgrade/sdn/egress-upgrade.feature [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    When I execute on the "<%= cb.pod1 %>" pod:
+      | curl | -s | --connect-timeout | 5 | <%= cb.ipecho_url %> |
+    Then the step should succeed 
+    And the output should contain "<%= cb.valid_ip %>"
+
+    # Remove label from egressip node and delete egress ip object
+    Given admin ensures "egressip" egress_ip is deleted after scenario
+    When I run the :get admin command with:
+      | resource       | node                               |
+      | l             | k8s.ovn.org/egress-assignable      |
```
Comment (pruan-rht) [inline features/upgrade/sdn/egress-upgrade.feature:132]: table alignment

Comment (huiran0826) [reply features/upgrade/sdn/egress-upgrade.feature]: Thanks, fixed it.

**Pull request merged**